src/query/api/v1/handler/prom/prom.go

```diff
@@ -63,6 +76,18 @@ func NewReadInstantHandler(opts Options, hOpts options.HandlerOptions) http.Hand
 	return newReadInstantHandler(opts, hOpts, queryable)
 }
 
+// DefaultReadRequestParser returns the default function that parse read request arguments
+func DefaultReadRequestParser(opts options.HandlerOptions) RequestParser {
```

Comment:
`hOpts`(for consistency with how `options.HandlerOptions` is called in another function).

Reply:
https://github.com/m3db/m3/pull/2884/commits/c70d0f506b1cc6b053e6fd086c5f70dc4aff3fe9#diff-1974feec4b84d1fc87b5ba59204e3cf6e3ac3b73fb77029186d60342a4323916R80